cli/tests/cli-reference@.md.snap

```diff
@@ -1,6 +1,7 @@
 ---
 source: cli/tests/test_generate_md_cli_help.rs
 description: "AUTO-GENERATED FILE, DO NOT EDIT. This cli reference is generated by a test as an `insta` snapshot. MkDocs includes this snapshot from docs/cli-reference.md."
+snapshot_kind: text
```

Comment:
is this one of those `insta` things again? :|

Reply:
That was probably it, regenerating...